tests/interfaces-cli/task.yaml

```diff
@@ -0,0 +1,35 @@
+summary: Check the interfaces command
+
+environment:
+  SNAP_NAME: network-consumer
+  SNAP_FILE: "./$[SNAP_NAME]_1.0_all.snap"
+  PLUG: network
+
+prepare: |
+  echo "Given a snap with the $PLUG plug is installed"
+  snapbuild ../lib/snaps/$SNAP_NAME .
```

Comment:
@kyrofa yeap, this PR requires #1367, which carries network-consumer, once that's in this should be fine

Thanks!